code/game/objects/items/devices/radio/radio.dm

```diff
@@ -33,6 +33,9 @@
 	var/use_command = FALSE  // If true, broadcasts will be large and BOLD.
 	var/command = FALSE  // If true, use_command can be toggled at will.
 
+	var/use_honken = FALSE
+	var/honken = FALSE  // same as command voice but HONK
```

Comment:
If this doesn't work on any headsets except the clown headset, then this should be entirely contained within the clown headset

Reply:
im futureproofing it incase i want to add a clown encryption key later